demo/src/main/java/com/bluelinelabs/conductor/demo/controllers/TransitionDemoController.java

```diff
@@ -56,9 +56,12 @@ public static TransitionDemo fromIndex(int index) {
         }
     }
 
-    @BindView(R.id.tv_title) TextView tvTitle;
-    @BindView(R.id.btn_next) FloatingActionButton btnNext;
-    @BindView(R.id.transition_root) View containerView;
+    @BindView(R.id.tv_title)
+    TextView tvTitle;
+    @BindView(R.id.btn_next)
+    FloatingActionButton btnNext;
+    @BindView(R.id.transition_root)
+    View containerView;
```

Comment:
Could we reformat these so the annotation and field are on the same lines again?

Reply:
Thanks, done.